spec/controllers/idv/verify_info_controller_spec.rb

```diff
@@ -349,6 +349,10 @@
               },
             ),
           )
+
+          event = @analytics.events["IdV: doc auth verify proofing results"].first
+          state_id = event[:proofing_results][:context][:stages][:state_id]
```

Comment:
good opportunity for `Hash#dig` if we want
```suggestion
          state_id = event.dig(:proofing_results, :context, :stages, :state_id)
```